packages/shopify-app-express/src/middlewares/validate-authenticated-session.ts

```diff
@@ -61,17 +82,6 @@ export function validateAuthenticatedSession({
         }
       }
 
-      const bearerPresent = req.headers.authorization?.match(/Bearer (.*)/);
-      if (bearerPresent) {
-        if (!shop) {
-          shop = await setShopFromSessionOrToken(
```

Comment:
Hey @paulomarg, I believe the removal of this code path introduced an issue #94. I have a comment there at the bottom explaining my particular situation, but there are a handful of other people with the issue there as well.

Without having much context, my guess is that we still need this fallback logic to set the `shop` variable when there are zero sessions in session storage (and no `shop` param in the request, of course).

Reply:
Sorry, I meant specifically the code below:
```
const payload = await api.session.decodeSessionToken(token);
shop = payload.dest.replace('https://', '');
```